sample/host/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "host",
+    "author": "Workleap",
+    "version": "0.0.0",
+    "description": "Host application to showcase @squide.",
+    "license": "Apache-2.0",
+    "type": "module",
+    "scripts": {
+        "dev": "webpack serve --config webpack.dev.js"
+    },
+    "devDependencies": {
+        "@pmmmwh/react-refresh-webpack-plugin": "0.5.10",
+        "@swc/core": "1.3.56",
+        "@types/react": "18.2.0",
+        "@types/react-dom": "18.2.0",
+        "@types/webpack": "5.28.0",
+        "@workleap/eslint-plugin": "1.2.0",
+        "@workleap/typescript-configs": "2.1.0",
+        "html-webpack-plugin": "5.5.1",
+        "react": "18.2.0",
```

Comment:
react/react-dom/react-router-dom/react-router should be in dependencies, not build dependencies